manifests/s/stacklok/minder/0.0.52/stacklok.minder.locale.en-US.yaml

```diff
@@ -0,0 +1,140 @@
+  * d4d1abade84f7a54794b3f6042058e784ea5b47c: build(deps): bump github.com/open-feature/go-sdk from 1.11.0 to 1.12.0 (#3457) (@dependabot[bot])
+  * c038374378e2c0166706f5e3d3874e964246b433: build(deps): bump github.com/openfga/cli from 0.4.0 to 0.4.1 in /tools (#3547) (@dependabot[bot])
+  * d8d7aa58e83d4f357c8f0dfdc08f622b4cb2b3a0: build(deps): bump github.com/openfga/go-sdk from 0.3.7 to 0.4.0 (#3469) (@dependabot[bot])
+  * b05c9d9a966837b7c093356c3741df6cd73c3dad: build(deps): bump github.com/openfga/openfga from 1.5.3 to 1.5.4 (#3515) (@dependabot[bot])
+  * b7c69085988a2db5a6ad01df7d5c7eec93d89ecc: build(deps): bump github.com/sigstore/sigstore-go from 0.3.0 to 0.4.0 (#3573) (@dependabot[bot])
+  * f96e0ea303192b54f2b1b098a34acf8e6da8be37: build(deps): bump github.com/spf13/viper from 1.18.2 to 1.19.0 (#3494) (@dependabot[bot])
+  * e84a8381bd73fd8977750befd93e646330db85d5: build(deps): bump github.com/stacklok/frizbee from 0.0.16 to 0.0.19 (#3572) (@dependabot[bot])
+  * dbe253dfd5fc86e4d4d69aaf8ad9d81e77fcef17: build(deps): bump github.com/styrainc/regal from 0.22.0 to 0.23.1 (#3574) (@dependabot[bot])
+  * f735fcda32f6bb09511be41729a465ac3262d376: build(deps): bump github/codeql-action from 3.25.6 to 3.25.7 (#3497) (@dependabot[bot])
+  * 4b993ce0e4ad26f96ec714d52fc50cc9072c17df: build(deps): bump github/codeql-action from 3.25.7 to 3.25.8 (#3527) (@dependabot[bot])
+  * 105c93cc04335380ed49f3583b99bf0bb4e92cbe: build(deps): bump github/codeql-action from 3.25.8 to 3.25.9 (#3594) (@dependabot[bot])
+  * 9ae8be6ef007fe6b73ee8da6570515d3e1a74a11: build(deps): bump github/codeql-action from 3.25.9 to 3.25.10 (#3620) (@dependabot[bot])
+  * 15fafdc655e2597a40da101020e5cb23e88bf326: build(deps): bump golang.org/x/crypto from 0.23.0 to 0.24.0 (#3524) (@dependabot[bot])
+  * 02bb768744ee98fde832ce6f91c671357f7de1d6: build(deps): bump golang.org/x/oauth2 from 0.20.0 to 0.21.0 (#3526) (@dependabot[bot])
+  * 37545331d533046a97f8ea0ce9d3b609aa9861be: build(deps): bump golang.org/x/term from 0.20.0 to 0.21.0 (#3525) (@dependabot[bot])
+  * 15af232d49a54b01b893edc2bc6f36915277365f: build(deps): bump golang.org/x/tools from 0.21.0 to 0.22.0 in /tools (#3529) (@dependabot[bot])
+  * b4abfe63c95e293444417d43fe174601a5d0a4eb: build(deps): bump google.golang.org/protobuf from 1.34.1 to 1.34.2 (#3585) (@dependabot[bot])
+  * bb173c2970bcc315139440cf6c2e28459fff3a2c: build(deps): bump google.golang.org/protobuf from 1.34.1 to 1.34.2 in /tools (#3583) (@dependabot[bot])
+  * ebc0bf54e02ccc2e866a6ace38537cc8e1595a0b: build(deps): bump google/clusterfuzzlite from 82652fb49e77bc29c35da1167bb286e93c6bcc05 to 884713a6c30a92e5e8544c39945cd7cb630abcd1 (#3545) (@dependabot[bot])
+  * ac2e877f87ff7ffd4c87919b8443624d3a36f717: build(deps): bump goreleaser/goreleaser-action from 5.1.0 to 6.0.0 (#3528) (@dependabot[bot])
+  * d31beeaf291ff98bb4b6f79e87cf821ee6296359: build(deps): bump k8s.io/client-go from 0.30.1 to 0.30.2 (#3596) (@dependabot[bot])
```

Comment:
```suggestion
```
Hi @stacklokbot, 

These release notes make the manifest too long. Is it okay if they're omitted?